### ISSUE TITLE
Add repo to linux package managers

### ISSUE THREAD

**Issue by rjsmith899** [staniel359/muffon#178]
When can Muffon be added to the Linux Mint, Debian, and PPA Ubuntu repositories? And can you add Muffon to Flatpak if it is not possible to add it to a repository? Add it to one of these repositories, or Flatpak can be more accessible to the user of the product to keep updated vs. manually doing it with a deb or app image. Thanks

**Comment by gamersi** [comment]
Hello @rjsmith899 
Thanks for your suggestion.
1. Debian Repo: This would need a dedicated maintainer which we don't have and don't have time to do ourselves. Also we would have to search for a *sponsor*.
2. PPA: It's sadly not *directly* possible to publish nodejs / electron applications to a PPA (Workaround: code another app which serves as a installer, which is time consuming and can introduce bugs)
3. flathub: That's actually possible, I'll talk to @staniel359 about it and keep you updated

Thanks for taking the time and giving muffon a shot, we really appreciate it.
\- gamer si
(Admin & Dev)

**Comment by rjsmith899** [comment]
Ok I didn't know how much work it would take for a repo. I am sorry about that. Flathub would be great it be host online. thanks for responding back quickly. I don't have no problems with the software. I just want to making some suggestion. I didn't know how much work it would take. thanks again

**Comment by xyloflake** [comment]
> Ok I didn't know how much work it would take for a repo. I am sorry about that. Flathub would be great it be host online. thanks for responding back quickly. I don't have no problems with the software. I just want to making some suggestion. I didn't know how much work it would take. thanks again

Suggestions are considered very valuable, thank you so much! The team is currently looking into it.

We have officially assigned @gamersi to this issue and he'll be taking care of this.

Have a great day :)

\-Xylo
(Admin & Dev)

**Comment by pinbuck** [comment]
I highly recommend and vouch flathub, as that's what linux users see when they search within "Software" on GNOME desktop or "Discover" on KDE desktop. The same can be said for Linux Mint I believe if you search in theirs.

**Comment by secretmango** [comment]
yes, a submission to flathub would be great. The distro repos are not thaat necessary.
